src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+$font-color-primary: #060b35;
+$font-color-collateral: #616070;
+$background-color:#fff;
+$border-color: #f3f3f3;
+$button-color:#00acdc;
+$hug: 16px;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
```

Comment:
99% of the time 
```
* {
  box-sizing: border-box;
}
```

Reply:
according to other Mate mentors' suggestions (M.Hrynko), we should avoid using `*{}`